docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,160 @@
+  * A neuron can be _non-dissolving_ which means that the timer is stopped and the neuron's dissolve delay remains the same. 
+  * A neuron can be _dissolving_ which means that the timer is decreasing the neuron's dissolve delay as time passes. 
+  * Once the timer is all the way down a neuron is _dissolved_ and the ICP tokens can be unstaked again.
+* **Age**: Every non-dissolving neuron has an age, which denotes how long it has been in the non-dissolving state. 
+A neuron's dissolve delay and age increases the neuron's voting power. 
+
+* **Maturity (positive number – percent)**:  When neurons vote, over time the NNS increases their so called maturity to reward them. Maturity can be converted into ICP by spawning (see below).
+
+### Interacting with neurons
+All interactions with a neuron go through the following API on the NNS governance cansiter. This includes everything in a neuron's lifetime such as the neuron's creation, modifications and voting, and finally unstaking the ICP tokens again from the neuron.
+```
+type ManageNeuron = record {
+  id : opt NeuronId;
+  command : opt Command;
+  neuron_id_or_subaccount : opt NeuronIdOrSubaccount;
+};
+```
+We will next to through a few important commands and how they can be used.
+
+
+### Creating a neuron and topping it up
+Staking ICP utility tokens in a neuron involves two commands:
+1. Sending ICP utility tokens to the neuron's subaccount.
+2. Claiming the neuron on the NNS governance canister, which basically means telling the governance canister that the transfer in 1. happened upon which NNS governance will create a new neuron.
+
+TODO: ADD DETAILS HOW THE SUBACCOUNT MUST BE COMPUTED!
+
+For the ledger transfer, the relevant `candid` interface can be found [here](https://github.com/dfinity/ic/blob/master/rs/rosetta-api/icp_ledger/ledger.did#L25) and contains the following details for a transaction:
+```
+type Transaction = record {
+    memo : Memo;
+    icrc1_memo: opt blob;
+    operation : opt Operation;
+    created_at_time : TimeStamp;
+}; 
+```
+To then claim the neuron, you can use the following command:
+TODO: WHICH WAY IS RECOMMENDED? ADD ARGUMENT EXPLANATION
+```
+type ClaimOrRefresh = record { by : opt By };
+type By = variant {
+  NeuronIdOrSubaccount : record {};
+  MemoAndController : ClaimOrRefreshNeuronFromAccount;
+  Memo : nat64;
+};
+type ClaimOrRefreshNeuronFromAccount = record {
+  controller : opt principal;
+  memo : nat64;
+};
+```
+Topping up an exising neuron with more tokens is called _refreshing_ a neuron and works in the same way. In a first step, additional tokens are sent to the neuron's account. In the second step, the NNS governance cansiter is informed so that it can update the neuron's stake after confirming with the ledger that there are new tokens. Note that refreshing of a neuron will also update the neuron's age to account for the fact that the newly added tokens have no age.
```

Comment:
```suggestion
Topping up an existing neuron with more tokens is called _refreshing_ a neuron and works in the same way. 

First, additional tokens are sent to the neuron's account. Then, the NNS governance canister is informed so that it can update the neuron's stake after confirming with the ledger that there are new tokens. 

:::info
Note that refreshing of a neuron will also update the neuron's age to account for the fact that the newly added tokens have no age.
:::
```